admin/app/controllers/authenticated/users/list.js

```diff
@@ -56,6 +65,10 @@ export default class ListController extends Controller {
   onChangeUsername(event) {
     this.usernameForm = event.target.value;
   }
+  @action
+  onChangeQueryType(event) {
+    this.queryTypeForm = event;
```

Comment:
Ici c'est une valeur qui est récupérée par la fonction et non un event, et donc il vaut mieux que cela soit reflété dans le code même si cela fonctionne car le nommage de l'argument n'a pas d'impact sur l'exécution du code : 

```suggestion
  onChangeQueryType(value) {
    this.queryTypeForm = value;
```